compute_tools/src/compute.rs

```diff
@@ -35,7 +35,7 @@ use crate::spec::*;
 /// Compute node info shared across several `compute_ctl` threads.
 pub struct ComputeNode {
     pub start_time: DateTime<Utc>,
-    pub connstr: String,
+    pub connstr: url::Url,
```

Comment:
I'd make a comment that it automatically escapes all needed url parts (I hope) with https://github.com/servo/rust-url/blob/master/percent_encoding/src/lib.rs

And when you call `.as_str()` it's already escaped